airbyte-integrations/bases/base-normalization/snowflake.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fishtownanalytics/dbt:0.21.1
+COPY --from=airbyte/base-airbyte-protocol-python:0.1.1 /airbyte /airbyte
```

Comment:
do we need to get this docker image anymore? can't we just `pip install airbyte-cdk` which contains the CDK models? not blocking but a good sweep

Reply:
is the COPY command equivalent to that pip install? (looks like the the docker image just contains `base_python_structs`)